### ISSUE TITLE
CP trying to overload combine/says no current combine 

### ISSUE THREAD

**Issue by farmertyler** [Courseplay/courseplay#5834]
im trying to figure out how to get the grain cart to offload the combine anything i try seems to not work i watch videos. it shows no combine in the current column not sure what to do/ is there a place i can look to see how to set this up properly?

**Comment by Tensuko** [comment]
#5272 there is also a video and https://github.com/Courseplay/courseplay/wiki

**Comment by farmertyler** [comment]
so like i have the course setup, i dont know why but why is there no automatic search for combine in "manage combines tab" it doesn't have a combine under current so, it has no idea whats going on.

**Comment by Tensuko** [comment]
logfile ?

**Comment by fekim71** [comment]
I have the same problem Courseplay 6.02.00047. WAITING FOR COMBINE TO CALL.  I have the lasted Courseplay file. I have tried deleting all courseplay files and mods. I started new games with new maps. I've done everything but  re-install the game. I figure a re-install wont work cause I play on a PC and a laptop with the same issue. It has got to be something with the newer versions of courseplay.  Because I went back to Courseplay 6.01.00330 on both machines and it works fine. 6.0.1.00330 works fine on both machines and I would be ok using 33 but it doesn't have course editor.
[log.txt](https://github.com/Courseplay/courseplay/files/5048382/log.txt)

![fsScreen_2020_08_09_15_01_17](https://user-images.githubusercontent.com/69436089/89741772-5155b900-da51-11ea-87ed-792cb1420041.png)
